session_security/tests/test_script.py

```
@@ -57,29 +64,32 @@ def test_activity_hides_warning(self):
-            except:
-                assert(False)  #Test fails if element invisibilty times out
-        except:
-            assert(False)  #Test fails if element visility times out
-
-
+                    expected_conditions.invisibility_of_element_located(
+                        (By.ID, "session_security_warning")
+                    )
+                )
+                self.assertFalse(el.is_displayed())
+            except Exception:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)